src/components/colorbar/draw.js

```diff
@@ -243,22 +245,38 @@ module.exports = function draw(gd, id) {
                 s.append('g').classed('cbtitleunshift',true)
                     .append('g').classed('cbtitle',true);
                 s.append('rect').classed('cboutline',true);
+                s.select('.cbtitle').datum(0);
             });
-        container.attr('transform','translate('+Math.round(fullLayout._size.l)+
-            ','+Math.round(fullLayout._size.t)+')');
+        container.attr('transform','translate('+Math.round(gs.l)+
```

Comment:
@alexcjohnson @mdtusz would you be opposed to adding a `translate` method to the  `d3.selection` prototype, like in [`d3-jetpack`](https://github.com/gka/d3-jetpack/blob/92987ce2b3aecba670e719d98694ca43bd713e95/d3-jetpack.js#L11-L15) ?